src/components/CampaignInitialisation/CampaignInitialisationScreen.tsx

```diff
@@ -112,6 +112,9 @@ export const CampaignInitialisationScreen: FunctionComponent<NavigationProps> =
         showErrorAlert(updateCampaignConfigError, () =>
           navigation.navigate("CampaignLocationsScreen")
         );
+      } else if (updateCampaignConfigError instanceof NetworkError) {
+        showErrorAlert(updateCampaignConfigError);
+        throw updateCampaignConfigError;
```

Comment:
What happens here actually? Will the error alert show up then immediately navigate to the ErrorBoundary?

Reply:
The error dialog will show up, with the background navigating to the `ErrorBoundary`.

Reply:
If the `ErrorBoundary` shows up, do we still need to call to show the errorAlert?